L1Trigger/L1THGCal/plugins/geometries/HGCalTriggerGeometryV9Imp2.cc

```diff
@@ -393,6 +493,10 @@ unsigned HGCalTriggerGeometryV9Imp2::getLinksInModule(const unsigned module_id)
   if (module_det_id.det() == DetId::HGCalHSc) {
     links = hSc_links_per_module_;
   }
+  if (module_det_id.det() == DetId::Forward && module_det_id.subdetId() == ForwardSubdetector::HFNose) {
+    links = 1;
```

Comment:
It should not be hardcoded. In particular if we use the "decentralized" ECON architecture (which is now the baseline), the number of links (elinks in that case) should be at least 2.

Reply:
yes, the links are the missing features.
to be added in the future, outside of this PR.